src/teleop/gui/src/components/ISHTask.vue

```diff
@@ -1,20 +1,53 @@
 <template>
 <div>
     <p>ISH</p>
+    <div class="box camera">
+      <Cameras v-bind:numCams="4" v-bind:primary="true" v-bind:mission="'Science'" v-bind:channel="'/cameras_control_ish'"/>
```

Comment:
the channel = /cameras_control_ish doesn't need to be here right

Reply:
also idk if this is what primary means but if this is what is being passed in for the service, then primary should be false. also idk if mission='science' is necessary either.

Reply:
Yeah these don't get uses in the Cameras.vue file, definitely not neccesary, and also yes primary should be false.